.github/workflows/yocto-build-deploy.yml

```diff
@@ -0,0 +1,663 @@
+name: "Yocto"
+
+on:
+  workflow_call:
+    secrets:
+      BALENA_API_KEY:
+        description: balena API key for deploying releases
+        required: false
+      S3_ACCESS_KEY:
```

Comment:
@klutchell we should be using ephemeral AWS access here, not storing/passing tokens/keys around.

Reply:
good call, we can look at how Flowzone does it

Reply:
setup of this AWS access:

https://github.com/balena-io/environments-base/pull/72
https://github.com/balena-io/environment-staging/pull/3284